### PR TITLE
Feature/11 ServiceResource DTO

### PR DESCRIPTION
**Pull request by MaryR-code**
Added DTO for ServiceResourse

### CODE REVIEW

**Review by yekaterinasavelyeva — APPROVED** [review]
It would be good to add some notOk test, because for now there are only "happy path" tests for service.

File: src/main/java/guru/bug/courses/srs/boundary/api/service/Service.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package guru.bug.courses.srs.boundary.api.service;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import guru.bug.courses.srs.entity.ServiceEntity;
+
+import java.util.UUID;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Service {
```
Comment (dima-fedotovs) [inline src/main/java/guru/bug/courses/srs/boundary/api/service/Service.java:9]: as far as I understand - this is a DTO - data transfer object. Usually, the name of the class should contain its belonging to this. Mean "ServiceDTO" or "ServiceDto"

**Review by dima-fedotovs — APPROVED** [review]
Seems ok. It is recommended to rename Service to ServiceDTO.

File: src/main/java/guru/bug/courses/srs/boundary/api/service/Service.java (hunk 1 above)
Comment (MaryR-code) [reply src/main/java/guru/bug/courses/srs/boundary/api/service/Service.java]: done

**Pull request merged**